src/compiler/diagnosticMessages.json

```diff
@@ -4165,5 +4165,17 @@
     "Generate 'get' and 'set' accessors": {
         "category": "Message",
         "code": 95046
+    },
+    "Convert arrow function": {

```

Comment:
nit. Though no one sees this message, i think it should this be `Add or remove braces in an arrow function`